pubspec.yaml

```diff
@@ -11,8 +11,16 @@ dependencies:
 dev_dependencies:
   unittest: '>=0.9.0 <0.12.0'
 transformers:
-  - serialization :
-      $include: ["test/transformer/test_models.dart"]
-  - serialization : 
-      $include: ["test/transformer/test_models_for_maps.dart"]
-      format: "maps"
+- serialization:
+    $include: ["test/transformer/test_models_for_maps.dart",
```

Comment:
It seems that one particular transformer can't be ran twice on the same asset. I get an error. It may be a bug in Barback. But basically to be able to run the transformer twice I had to make sure they tun on a completely different set of files via this $include and $exclude. Looks like a bug (or "limitation") of transformers... but I had no choice.